Users.js

```diff
@@ -72,19 +80,56 @@ class Users extends React.Component {
       path: 'users',
       GET: {
         params: {
-          query: makeQueryFunction(
-            'username=*',
-            'username="$QUERY*" or personal.firstName="$QUERY*" or personal.lastName="$QUERY*" or personal.email="$QUERY*" or barcode="$QUERY*" or id="$QUERY*" or externalSystemId="$QUERY*"',
-            {
-              Status: 'active',
+          query: (...args) => {
+            /*
+              This code is not DRY as it is copied from makeQueryFunction in stripes-components.
+              This is necessary, as makeQueryFunction only referneces query paramaters as a data source.
+              STRIPES-480 is intended to correct this and allow this query function to be replace with a call
```

Comment:
I wonder if it would be better to do STRIPES-480 before merging this. Presumably the result would be a smaller and more pleasant change-set.

Reply:
I still feel this is worth fixing. At present, the inclusion of a vast chunk of modified makeQueryFunction overwhelms the rest of this change-set and makes it hard to see what's really changing.